tutorials/preprocessing/25_background_filtering.py

```diff
@@ -66,16 +66,16 @@
 .. math::
    :label: summations
 
-    y(n) &= b_0 x(n) + b_1 x(n-1) + \ldots + b_M x(n-M)
-            - a_1 y(n-1) - a_2 y(n - 2) - \ldots - a_N y(n - N)\\
+    y(n) &= b_0 x(n) + \ldots + b_M x(n-M)
+            - a_1 y(n-1) - \ldots - a_N y(n - N)\\
          &= \sum_{k=0}^M b_k x(n-k) - \sum_{k=1}^N a_k y(n-k)
 
 In other words, the output at time :math:`n` is determined by a sum over
 
-    1. the numerator coefficients :math:`b_k`, which get multiplied by
-       the previous input values :math:`x(n-k)`, and
-    2. the denominator coefficients :math:`a_k`, which get multiplied by
-       the previous output values :math:`y(n-k)`.
+1. the numerator coefficients :math:`b_k`, which get multiplied by the previous
+input values :math:`x(n-k)`, and
+2. the denominator coefficients :math:`a_k`, which get multiplied by the
+previous output values :math:`y(n-k)`.
```

Comment:
Will probably fix it (to be tested)
```suggestion
1. the numerator coefficients :math:`b_k`, which get multiplied by the previous
   input values :math:`x(n-k)`, and
2. the denominator coefficients :math:`a_k`, which get multiplied by the
   previous output values :math:`y(n-k)`.
```